src/common.js

```diff
@@ -409,3 +409,9 @@ function _register(name, implementation, wrapOptions) {
   }
 }
 exports.register = _register;
+
+// Expose the path to the NodeJS binary to utilities
+var isElectron = Boolean(process.versions.electron);
+if (!isElectron) {
+  exports.execPath = process.execPath;
```

Comment:
Probably the better spot for it. We'll have to be careful with the name we come up for in #534 (which is a different feature, that currently sounds very similar).